docker-compose.yml

```diff
@@ -6,7 +6,14 @@ services:
     container_name: sdk-harness-algod
     build:
       context: .
-      dockerfile: ./docker/algod/Dockerfile
+      dockerfile: "./docker/algod/${TYPE}/Dockerfile"
```

Comment:
You could use this to set a default for type

```
${TYPE:-channel}
```

Reply:
Nice, `docker-compose up` this will let `docker-compose up` with no scripts to work again.